lib/routes.php

```diff
@@ -15,9 +15,12 @@
 
 // /{pagename}.{representation}
 if ($representation != '') {
-  if (!$page_exists) {
+  if (!$page_exists) {	  
     header('HTTP/1.0 404 Not Found');
-    die("Page not found: <a href=".EDITH_URI."/".$page->name.">Create me!</a>");
+    
+    // generate 404 response by utilizing my 404 page (html)
```

Comment:
I think we can remove this comment since it doesn’t add to what the underlying lines do.

```suggestion
```

Reply:
Yup, that was just for my reference